mobile/library/objective-c/EnvoyConfiguration.mm

```diff
@@ -177,6 +177,11 @@ - (instancetype)initWithAdminInterfaceEnabled:(BOOL)adminInterfaceEnabled
   builder.enableGzipDecompression(self.enableGzipDecompression);
   builder.enableBrotliDecompression(self.enableBrotliDecompression);
 
+  for (NSString *key in self.runtimeGuards) {
+    BOOL value = [[self.runtimeGuards objectForKey:key] isEqualToString:@"true"];
```

Comment:
should we apply false's too? For cases when default is equal to `true` and we want to override with `false`

Reply:
This applies true and false equally, as verified by the tests.

If you call `.setRuntimeGuard("abc", false)` then that gets mapped to an NSDictionary of `@{@"abc", @"false"}`, which means that `[[self.runtimeGuards objectForKey:@"abc"] isEqualToString:@"true"];` produces a result of `NO`, which passes to the C++ builder as `builder.setRuntimeGuard("abc", false)`.